src/transformers/models/umt5/modeling_umt5.py

```diff
@@ -481,6 +501,7 @@ class UMT5PreTrainedModel(PreTrainedModel):
     supports_gradient_checkpointing = True
     _no_split_modules = ["UMT5Block"]
     _keep_in_fp32_modules = ["wo"]
+    _supports_cache_class = True
```

Comment:
This enables the `test_new_cache_format` test -> converting back and forth between the new cache and the legacy cache with cross attention is tested